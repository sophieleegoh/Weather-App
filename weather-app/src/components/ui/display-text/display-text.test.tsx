import renderer from 'react-test-renderer';
import DisplayText from './index';

describe("Display text snapshot tests", () => {
  it("Renders the component", () => {
    const component = renderer.create(
      <DisplayText text="This is some test display text" />
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  })
})