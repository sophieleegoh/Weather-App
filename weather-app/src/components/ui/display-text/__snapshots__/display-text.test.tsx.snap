// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display text snapshot tests Renders the component 1`] = `
<p
  className="py-10"
>
  This is some test display text
</p>
`;
